clang/test/SemaSYCL/unsupported_math.cpp

```diff
@@ -0,0 +1,26 @@
+// RUN: %clangxx --sycl %s -c -o a_kernel.bc -Xclang -verify
+
+#include <CL/sycl.hpp>
+
+int run_test_a(int v, cl::sycl::queue &deviceQueue) {
+  try {
+    int arr[] = {v};
+    {
+      cl::sycl::buffer<int, 1> buf(arr, 1);
```

Comment:
Please, reduce the test case to minimal reproducer.
Take a look at this example: https://github.com/intel/llvm/blob/sycl/clang/test/SemaSYCL/bool-vectors.cpp.

This test should not require SYCL objects like queue and buffer.

Reply:
I reduced the test to a minimal reproducer as asked. Thanks